packages/speech/src/index.js

```diff
@@ -87,6 +88,9 @@ function Speech(options) {
   };
 
   common.GrpcService.call(this, config, options);
+  this.api = {
```

Comment:
I figure other packages can follow this same convention, (nest autogen services under `api`) as some packages will actually need to instantiate multiple autogen services, i.e. Pub/Sub.